docs/examples/index.rst

```diff
@@ -0,0 +1,10 @@
+=====================================================
```

Comment:
I think that this should be where we have some more generic examples that showcase the broad range of things that can be done across the SDK. The files in `docs/examples` should go under `docs/pma/examples` and used in the PMA docs specifically, as that's where they are relevant.

I think that the high-level examples should show how someone might be able to connect the different subsystems together (e.g. how to show the output of a PMA component value onto the OLED)